phone-scripts/01-create-sudo-user.sh

```diff
@@ -13,4 +13,5 @@ infecho () {
 infecho "Adding user \"pine\"..."
 adduser pine
 passwd pine
-usermod -aG wheel,video pine
+groupadd feedbackd
+usermod -aG wheel,video,feedbackd pine
```

Comment:
Future versions of some purism software will be using feedbackd instead of video for accessing some feedbackd related events.